ConservationInternational/resilienceatlas
# RA2-200: Adding description to Layer serializer and reference to Source serializer

Title of this PR is probably self-explaining ;). This PR just adds description to Layer serializer and reference to Source serializer so they can be used by FE as replacement for `info` attribute

## Testing instructions

Open Swagger doc and download layers information about API. Check that `description` is there for Layer and if layer has sources, there is also `reference` attribute.

## Tracking

https://vizzuality.atlassian.net/browse/RA2-200
